libbeat/version/helper.go

```diff
@@ -21,6 +21,11 @@ import "time"
 
 // GetDefaultVersion returns the current libbeat version.
 // This method is in a separate file as the version.go file is auto generated
+//
+// TODO(AndersonQ): does it need to be changed? in the agent's code there is
+// still a GetDefaultVersion with the same behaviour as this one and it's used
+// in several places. Also, in the agent code, there is another function for
+// getting the package version.
```

Comment:
This seems to be pretty widely used, I think we need the version returned from here to also be the agent package version.

https://github.com/search?q=repo%3Aelastic%2Fbeats%20GetDefaultVersion(&type=code

I see it setting the user agent used in network requests for some inputs for example.